src/panels/config/automation/trigger/types/ha-automation-trigger-state.ts

```diff
@@ -182,7 +199,22 @@ export class HaStateTrigger extends LitElement implements TriggerElement {
       return false;
     }
     try {
-      assert(this.trigger, stateTriggerStruct);
+      assert(
+        this.trigger,
+        refine(
+          stateTriggerStruct,
+          "one each of: to/not_to, from/not_from",
+          (value) => {
+            if ("to" in value && "not_to" in value) {
+              return "bad";
```

Comment:
I thought that maybe this string being returned would be part of the error message, but it seems to never actually appear anywhere? 🤷‍♂️ 

Anyway this is a relatively unimportant and strange case.
![image](https://github.com/user-attachments/assets/d828dc41-12e9-438a-a078-1a2bd318c314)